src/stories/AmplienceImageStudio.stories.ts

```diff
@@ -6,7 +6,8 @@ import {
 } from '../AmplienceImageStudio';
 import { ImageStudioReason, SDKImage } from '../types';
 
-const IMAGE_STUDIO_URL = 'https://app.amplience.net/image-studio/';
+const IMAGE_STUDIO_DOMAIN = 'https://app.amplience.net';
+// const IMAGE_STUDIO_DOMAIN = 'http://localhost:5173';
```

Comment:
Is this line still needed?

Reply:
kinda, kinda-not - its helpful to just un/comment when running storybook local and pointing to a local `image-studio` deployment, theres defo cases where you'd wanna tests against both so i left it in for ease - mostly because i keep removing it to check changes in for this file, then have to re-instate it 🙈